library/src/main/java/com/novoda/downloadmanager/MigrationJob.java

```diff
@@ -41,35 +49,51 @@ public void run() {
         SqlDatabaseWrapper database = new SqlDatabaseWrapper(sqLiteDatabase);
 
         FilePersistence filePersistence = FilePersistenceCreator.newInternalFilePersistenceCreator(context).create();
+        String basePath = filePersistence.basePath().path();
```

Comment:
can you move this near line 77?

Reply:
I didn't do this because I wanted to keep the creation of dependencies together, away from where the execution happens.

Reply:
I wasn't sure where that was used... this is why I asked